source/sync/configure/sync-settings.txt

```diff
@@ -61,15 +61,64 @@ update the backend schema by changing the client model.
 To learn more about how the Realm Object Schemas map to the App Services Schemas
 when using Development Mode, refer to :ref:`Data Model Mapping <sync-data-model-mapping>`.
 
-For more information about modifying synced object schemas, including how to
-handle breaking changes, see: :ref:`<synced-schema-overview>`.
+For more information about modifying synced object schemas, see:
+:ref:`<synced-schema-overview>`.
 
 .. important:: Disable Development Mode for Production Apps
    
    Development mode is a development utility that is not suitable for 
    production use. Make sure that you turn off Development Mode before you 
    make your app accessible in a production environment.
 
+.. _dev-mode-breaking-changes:
+
+Breaking Changes
+````````````````
+
+*Applies to App Services Apps created after August 9th, 2023.* 
+
+App Services Apps in Development Mode that were created after August 9th, 2023
+can make :ref:`breaking changes <breaking-change-quick-reference>` from their
+client code to synced object schemas.
+
+.. note:: Apps created before August 9th, 2023
+   
+   For Apps created before August 9th, 2023, you need to update your
+   object schema in the App Services UI. For details, see
+   :ref:`<synced-schema-overview>`
+
+To make a breaking change from client code:
+
+#. Change your local object model and let it sync to your App.
+#. Delete your local realm and data.
+#. Open a new realm with your updated object model.
+
+To delete a Realm file use the Realm SDK-specific methods:
```

Comment:
If the requirement is to delete the realm, there are multiple ways we might do that. If an app is in development, presumably we can just delete the App from the simulator. Also, there are some considerations about _when_ you can use the `.delete` method in Swift. (And this delete method isn't correct for Swift - it's `Realm.deleteFiles(for: config)`)

Instead of manually supplying untested SDK methods here without helpful context/API reference links, maybe it makes sense to link out to the SDK Delete pages? i.e. https://www.mongodb.com/docs/realm/sdk/swift/realm-files/delete-a-realm/

Reply:
Yeah... not all of the SDKs document how to delete a realm. Like the JS SDKs. *hides*

I admittedly opted for the lazier approach with doing the table of untested code snippets. Instead, it probably makes sense to link to SDK docs where it's documented and link to API reference docs where it's not. I'll update!

Reply:
I guess I know some pages that should be added to the "gaps" epic! 😂 

This seems like a reasonable compromise.